.goreleaser.yaml

```diff
@@ -0,0 +1,138 @@
+# This is an example .goreleaser.yml file with some sensible defaults.
+# Make sure to check the documentation at https://goreleaser.com
+before:
+  hooks:
+    # You may remove this if you don't use go modules.
+    - go mod tidy
+    # you may remove this if you don't need go generate
+    - go generate ./...
+builds:
+  - id: daje
+    binary: daje
+    main: .
+    goos:
+      - linux
+      - darwin
+      # - windows
+    goarch:
+      - amd64
+      - arm64
+    env:
+      - CGO_ENABLED=0
+      - CC_darwin_amd64=o64-clang
+      - CCX_darwin_amd64=o64-clang+
+      - CC_darwin_arm64=aarch64-apple-darwin20.2-clang
+      - CCX_darwin_arm64=aarch64-apple-darwin20.2-clang++
+      # - CC_windows_amd64=x86_64-w64-mingw32-gc
+      # - CCX_windows_amd64=x86_64-w64-mingw32-g++
+      - 'CC={{ index .Env (print "CC_" .Os "_" .Arch) }}'
+      - 'CCX={{ index .Env (print "CCX_" .Os "_" .Arch) }}'
```

Comment:
Can you separate Linux and macOS for the build? Apart that I do not think we need all those env, I'll suggest to keep only the `CGO_ENABLED=0`

Reply:
Those vars are needed for different architectures, for Linux and mac.
I could make different builds for mac and Linux, but doesn't get the point to do that.